edge-ai-void-detection/modules/captureimages/Dockerfile.amd64

```diff
@@ -11,6 +11,7 @@ RUN pip install --upgrade pip
 
 COPY requirements.txt ./
 RUN pip install -r requirements.txt
+RUN pip install requests
```

Comment:
Maybe it would make sense to put `requests==2.7.0` within `requirements.txt` rather than directly within the `Dockerfile`

Reply:
Great suggestion, @richardjortega, thank you!
Pushed and squashed.